src/components/Navbar.js

```diff
@@ -0,0 +1,46 @@
+import { NavLink, useLocation } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import AddLink from './AddLink'
+import medicationsDisabled from '../icons/pill-disabled.svg'
+import medicationsEnabled from '../icons/pill-enabled.svg'
+import observationsDisabled from '../icons/observations-disabled.svg'
+import observationsEnabled from '../icons/observations-enabled.svg'
```

Comment:
Can these not be styled through differently props? In that case you'd have to use the svg instead of an image. Might be a good idea since you already have the `svg`.